vendor/github.com/portworx/sched-ops/k8s/k8s.go

```diff
@@ -930,6 +934,28 @@ func (k *k8sOps) DescribeDeployment(depName, depNamespace string) (*apps_api.Dep
 	return &dep.Status, err
 }
 
+func (k *k8sOps) UpdateDeployment(deployment *apps_api.Deployment, newRepl *int32) error {
```

Comment:
lets' keep this generic and only take the updated deployment object (which will have the updated replica set number